domains/transfer/Transfer.API/API/Shared/Migrations/20231123093303_AddTransferAgreementProposalsAndDeleteConnections.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace API.Shared.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddTransferAgreementProposalsAndDeleteConnections : Migration
```

Comment:
Should there be a migrations test here?

Reply:
I don't think so since we only are deleting and adding tables